community/sinit/files/etc/rc.d/sinit.runit.boot

```diff
@@ -0,0 +1,5 @@
+busybox_runit () {
+    sh -c 'ubase-box respawn /usr/bin/runsvdir -P /var/service' &
```

Comment:
```sh
ubase-box respawn /usr/bin/runsvdir -P /var/service &
```

Reply:
@dylanaraps  we might run into job control issues with getty and this by dropping sh -c. This needs testing.

Reply:
I've seen weird things happen when messing with these parts. Has to to do with the way we fork the process

Reply:
ok it's tested this is fine